app/src/main/java/ch/admin/bag/dp3t/updateboarding/UpdateBoardingActivity.java

```diff
@@ -0,0 +1,46 @@
+package ch.admin.bag.dp3t.updateboarding;
+
+import android.content.Intent;
+import android.os.Bundle;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.FragmentActivity;
+
+import org.dpppt.android.sdk.DP3T;
+
+import ch.admin.bag.dp3t.R;
+
+public class UpdateBoardingActivity extends FragmentActivity {
+
+	// Increment this number for each new Update Boarding
+	public static final int UPDATE_BOARDING_VERSION = 1;
```

Comment:
Could we just use the app's version code instead?

Reply:
If we use the App Version Code, the update boarding would be shown for every app update and we would have to disable it manually if we **don't want** to show an update boarding for a small bug fix update for example. With this solution we have to manually increase this number if we **want** an update boarding to be shown, which is the better solution in my eyes because in this case we have to modify this file anyways. 

Or did I misunderstand your comment?

Reply:
yea, no you're probably right, having a custom number gives more flexibility.